apps/iotune/iotune.cc

```diff
@@ -104,6 +104,7 @@ struct evaluation_directory {
     unsigned _force_io_depth;
     uint64_t _available_space;
     uint64_t _min_data_transfer_size = 512;
+    uint64_t _max_iop_size = 256 * 4096;
```

Comment:
Despite patch commit assumes, this value is pure abstract in a sense that -- iotune still uses some _other_ request size for its bandwidth tests. Am I wrong?

Reply:
> Despite patch commit assumes, this value is pure abstract in a sense that -- iotune still uses some other request size for its bandwidth tests. Am I wrong?

Yes you are (partially) right, iotune still submits 1MB requests unconditionally. It's at the block layer where requests will then get split up into multiple requests as per this value.

We should probably change iotune to respect this in any case but I didn't want to overload this PR initially.